### PR TITLE
DUOS-1042 Added Election Status Output Processing to NewChairConsole

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [DUOS-1042](https://broadworkbench.atlassian.net/browse/DUOS-1042)

PR adds processing for election status so that ```Open``` and ```Final``` statuses can have customized outputs. ```Open``` will show ```votes submitted / votes expected``` whereas ```Final``` will display ```Accepted``` or ```Denied``` based on the final vote itself. Helpers have been implemented on search to reflect visible data. 

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] PR is labeled with a Jira ticket number and includes a link to the ticket
- [x] PR is labeled with a security risk modifier [no, low, medium, high] 
- [x] PR describes scope of changes

In all cases:

- [x] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [x] Get PO sign-off for all non-trivial UI or workflow changes
- [x] Verify all tests go green
- [x] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/NewChairConsole.js [hunk 1]
```diff
@@ -10,6 +10,35 @@ import PaginationBar from '../components/PaginationBar';
...
+  return !isNil(targetDate);
+};
+
+const wasFinalVoteSubmitted = (voteData) => {
+  const {type, vote} = voteData;
+  //vote status capitalizes final, election status does not
+  return type === 'FINAL' && !isNil(vote);
+};
+
+const processElectionStatus = (election, votes) => {
```
Comment (JVThomas) [inline src/pages/NewChairConsole.js:26]: I think the default max value for cyclometric complexity is too low. Most teams use 8-10 as their max value. Is it okay for me to ignore this issue? (You can see the Codacy issue on the files changed tab).

Comment (rushtong) [reply src/pages/NewChairConsole.js:26]: Absolutely!

Comment (rushtong) [reply src/pages/NewChairConsole.js:26]: In fact, this issue is already ignored in codacy, but it still reports as a finding for some reason so this looks like a codacy bug.

Comment (JVThomas) [reply src/pages/NewChairConsole.js:26]: I re-ran the Codacy action (Bernick gave me access) so the code should be good for review.

**Review by rushtong — APPROVED** [review]
👍🏽

**Comment by JVThomas** [comment]
Leaving a note that the audit flag on the immer dependency (for react-scripts) is a false positive
https://github.com/facebook/create-react-app/pull/10412#issuecomment-781391257

**Comment by JVThomas** [comment]
Requesting a re-review since I've made corrections to the vote count processing step.

**Review by rushtong — APPROVED** [review]
Changes look reasonable 👍🏽

**Pull request merged**